google/cloud/storage/object_metadata_test.cc

```diff
@@ -442,6 +442,15 @@ TEST(ObjectMetadataTest, SetEventBasedHold) {
   EXPECT_NE(expected, copy);
 }
 
+/// @test Verify we can change the storageClass field.
+TEST(ObjectMetadataTest, SetStorageClass) {
+  auto expected = CreateObjectMetadataForTest();
+  auto copy = expected;
+  copy.set_content_type("NEARLINE");
```

Comment:
Shouldn't this be `set_storage_class()` and `copy.storage_class()`?

Reply:
Indeed. Fixed.